fire/__main__.py

```diff
@@ -0,0 +1,16 @@
+"""
+Module for running python fire as a "main" function and thereby allowing using
+fire with other programs.
+"""
+import importlib
+import sys
+
+import fire
+
+def main(args):
+  module_name = args[1]
+  module = importlib.import_module(module_name)
+  fire.Fire(module, name=module_name, command=args[2:])
```

Comment:
Do we want `module` or `vars(module)`?

Reply:
I was hoping that vars(module) would make it so that interactive mode would look like `from module import *`, but that didn't work either.

My inclination would be to leave it as just `module` - seems like the inference should be for how python objects work by default, not for dictionaries (i.e., in terms of hiding "protected" attributes, etc).

Reply:
Oh yeah, my bad. I was also making that suggestion hoping it would behave like "import *".
Leaving it as `module` sgtm.